tools/run_tests/run_xds_tests.py

```diff
@@ -2238,6 +2238,34 @@ def test_csds(gcp, original_backend_service, instance_group, server_uri):
                        test_csds_timeout_s)
 
 
+def maybe_write_sponge_properties():
+    """Writing test infos to enable more advanced testgrid searches."""
+    if 'KOKORO_ARTIFACTS_DIR' not in os.environ:
+        return
+    if 'KOKORO_GITHUB_COMMIT_URL' not in os.environ:
+        return
+    # Commit url example:
+    # KOKORO_GITHUB_COMMIT_URL="https://github.com/lidizheng/grpc/commit/e38cbb245a69b3112bb65b164ce2ca6b2dff0d7b"
```

Comment:
I guess this'll work... But I'm not 100% sure this `/commit/` format is guaranteed. Could this be a tag?
Would it be too much of a hustle to calculate it in the build script and pass as an env var? Something like
```
-e "GIT_ORIGIN_URL="$(git remote get-url origin)" \
```

Then just printing it in the script?

Reply:
Done as suggested.